index.js

```diff
@@ -1,49 +1,65 @@
-const axios = require('axios');
-const FormData = require('form-data');
-const codec = require('string-codec');
+const axios = require("axios");
+const FormData = require("form-data");
+const codec = require("string-codec");
+const cron = require("node-cron");
 
 const prepForm = (obj) => {
   const { long, lat, desc, cookies, isCheckOut = false } = obj;
   const data = new FormData();
-  const status = (isCheckOut) ? 'checkout' : 'checkin';
+  const status = isCheckOut ? "checkout" : "checkin";
 
-  const longEncoded = codec.encoder(codec.encoder(long,'base64'),'rot13');
-  const latEncoded = codec.encoder(codec.encoder(lat,'base64'),'rot13');
+  const longEncoded = codec.encoder(codec.encoder(long, "base64"), "rot13");
+  const latEncoded = codec.encoder(codec.encoder(lat, "base64"), "rot13");
 
-  data.append('longitude', longEncoded);
-  data.append('latitude', latEncoded);
-  data.append('status', status);
-  data.append('description', desc);
+  data.append("longitude", longEncoded);
+  data.append("latitude", latEncoded);
+  data.append("status", status);
+  data.append("description", desc);
 
   const config = {
-    method: 'post',
-    url: 'https://hr.talenta.co/api/web/live-attendance/request',
-    headers: { 
-      'Cookie': cookies, 
-      ...data.getHeaders()
+    method: "post",
+    url: "https://hr.talenta.co/api/web/live-attendance/request",
+    headers: {
+      Cookie: cookies,
+      ...data.getHeaders(),
     },
-    data : data
+    data: data,
   };
 
   return config;
 };
 
+const parseTime = (time) => {
+  return time.split(":");
+};
+
 const attendancePost = async (obj) => {
   const config = prepForm(obj);
   const resp = await axios(config);
 
   return resp.data;
-}
+};
+
+const scheduler = async (time, callback) => {
+  const [hour, min] = parseTime(time);
+
+  const task = cron.schedule(`${min} ${hour} * * 1-5`, async () => {
```

Comment:
okay, nice to have scheduler method from modules.
then executed from another examples (scheduler.js)

Reply:
but since you just called a callback, why don't you move it to scheduler.js for complete method?
But this is also works, but I prefer scheduler is on schedulers unless specific method needed from API related here

Reply:
alright, so we just need to move scheduler function into scheduler.js alongside with time parser, right?